src/platform_linux/platform.c

```diff
@@ -372,7 +372,7 @@ platform_assert_msg(platform_log_handle *log_handle,
                     const char          *message,
                     va_list              varargs)
 {
-   static char assert_msg_fmt[] = "OS-pid=%d, OS-tid=%lu, Thread-ID=%lu, "
+   static char assert_msg_fmt[] = "OS-pid=%d, OS-tid=%d, Thread-ID=%lu, "
```

Comment:
Compilation on L299 (in the definition of `platform_error_log()`) fails if this is specified as `%lu`. I found that `sizeof(pid_t)` is `int`, so `%d` is the right specifier.

This old code succeeded compilation as this print-format specifier is hidden behind a collection of nested macros so isn't visible actually till run-time.